virttest/utils_test/libvirt.py

```diff
@@ -1027,22 +1027,23 @@ def pre_pool(self, pool_name, pool_type, pool_target, emulated_image,
                 source_host = self.params.get("source_host", "localhost")
                 extra = "--source-host %s --source-path %s" % (source_host,
                                                                nfs_path)
-        elif pool_type == "iscsi":
+        elif pool_type == "iscsi" or pool_type == "iscsi-direct":
```

Comment:
suggest update 'if' condition to 'if pool_type in ["iscsi", "iscsi-direct"]'.

Reply:
Updated